backend/communications/mongo_models.py

```diff
@@ -0,0 +1,43 @@
+from mongoengine import Document, StringField, DateTimeField, ListField, ReferenceField, CASCADE
+import datetime
+
+class Room(Document):
+    """
+        Represents a chat room between one or more participants.
+    """
+    meta = {
+        'db_alias': 'chat_db',
+        'collection': 'rooms',
+        'indexes': ['name', 'participants']
+    }
+
+    name = StringField(required=True, unique=True)
+    participants = ListField(StringField(), default=[])
+    created_at = DateTimeField(default=datetime.datetime.utcnow)
+    updated_at = DateTimeField(default=datetime.datetime.utcnow)
+
```

Comment:
можемо надписати метод `save` щоб `updated_at` поле обновлялось за замовчуванням.

```suggestion
    def save(self, *args, **kwargs):
        # auto-update timestamp
        self.updated_at = datetime.datetime.utcnow()
        return super().save(*args, **kwargs)
        ```

Reply:
Done